stable/pipelines/templates/argo/deployment.yaml

```diff
@@ -0,0 +1,59 @@
+{{- if .Values.deployment.create }}
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  labels:
+    app: {{ template "pipelines.name" . }}
+    chart: {{ template "pipelines.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  name: workflow-controller
+  namespace: {{ .Release.Namespace }}
+spec:
+  progressDeadlineSeconds: 600
+  replicas: 1
```

Comment:
progressDeadlineSeconds, and replicas look like they should be defined and overridable via `values.yaml`

Reply:
done